src/html/translate-html-parser.ts

```diff
@@ -70,7 +70,7 @@ export  default class TranslateHtmlParser implements htmlparser.Handler {
                 translatedAttributeName = attributeName.substr("translate-attr-".length),
                 defaultText = attributes["translate-default-attr-" + translatedAttributeName];
 
-            this.registerTranslation(translationId, defaultText);
+            this.registerTranslation(translationId, attributes["translate-default"] || defaultText);
```

Comment:
This is incorrect. If you define one translate-default but the element has multiple translated attributes (translate-attr) elements, then all elements would have the same default text. Therefore, the translate-default-attr-title attribute should be used to define the default text

```html
<a translate translate-attr-title="TITLE-ID" translate-default-attr-title="Default for title attr" href="#"><i class="fa-home fa" /></a>
```